core/src/main/java/tech/pegasys/web3signer/core/metrics/Web3SignerMetricCategory.java

```diff
@@ -23,9 +25,7 @@
 import org.hyperledger.besu.plugin.services.metrics.MetricCategory;
 
 public enum Web3SignerMetricCategory implements MetricCategory {
```

Comment:
Should this be renamed to e.g. HttpMetricCategory?

Reply:
Will rework this in another PR